scompose/client/check.py

```diff
@@ -25,10 +25,10 @@ def main(args, parser, extra):
 
     # validate compose files
     for f in args.file:
-        result = validate_config(f)
-        if not result and not args.preview:
+        valid = validate_config(f)
+        if valid and not args.preview:
```

Comment:
What about the case when it's valid and a preview?

Reply:
Hang on... I messed up the files to be commited... Give 1 sec, I will open another pr

Reply:
True... I just didn't want that to have "add_deploy_conf_group" branch name :)

Reply:
> Sure thing! Also if I remember correctly, json schema just returns None if it's valid. So maybe we should more explicitly turn that into a True/False so it's clear, e.g., instead of:
> 
> ```python
> return validate(thing, schema=schema)
> ```
> 
> ```python
> return validate(thing, schema=schema) is None
> # True == valid
> # False == not valid (or might raise error I can't remember!)
> ```

The problem with the existing impl is that if validate fails then an exception would be thrown...so the message "file is invalid" followed by exit code 1 wouldn't happen the way it was expected on https://github.com/singularityhub/singularity-compose/blob/master/scompose/client/check.py#L29-L32

Let's continue this discussion on #55 :)